src/Utils/types/data/JsonCommunicationType.ts

```diff
@@ -0,0 +1,27 @@
+type TypeData = 'element' | 'array' | 'status';
+
+type InfoArrayData = {
```

Comment:
Skoro niżej są interfejsy, to i tu można. Btw. czy te typy nie są przypadkiem kopiowane na FE?